src/io/flutter/inspector/DiagnosticsNode.java

```diff
@@ -620,4 +616,33 @@ private static Icon getCustomIcon(String text) {
 
     return iconMaker.getCustomIcon(text, isPrivate ? CustomIconMaker.IconKind.kMethod : CustomIconMaker.IconKind.kClass);
   }
+
+  /**
+   * Returns true if two diagnostic nodes are indistinguishable from
+   * the perspective of a user debugging.
+   * <p>
+   * In practice this means that all fields but the objectId and valueId
+   * properties for the DiagnosticsNode objects are identical. The valueId
+   * field may change even for properties that have not changed because in
+   * some cases such as the 'created' property for an element, the property
+   * value is created dynamically each time 'getProperties' is called.
+   */
+  public boolean identicalDisplay(DiagnosticsNode node) {
+    if (node == null) {
+      return false;
+    }
+    Set<String> keys = json.keySet();
```

Comment:
@jacob314, this API doesn't exist in 2017.1 and 2017.2; see https://travis-ci.org/flutter/flutter-intellij/builds/336668158?utm_source=github_status&utm_medium=notification

The build was broken for pub reasons, so it masked this additional failure, but we should re-run travis in those cases to get to green builds :)